src/useRoom.ts

```diff
@@ -26,30 +25,24 @@ export interface RoomState {
   error?: Error;
 }
 
-export interface RoomOptions {
-  sortParticipants?: (participants: Participant[]) => void;
-}
-
-export function useRoom(options?: RoomOptions): RoomState {
-  const [room, setRoom] = useState<Room>();
+export function useRoom(roomOptions?: RoomOptions): RoomState {
+  const [currentRoom, setCurrentRoom] = useState<Room>();
   const [isConnecting, setIsConnecting] = useState(false);
   const [error, setError] = useState<Error>();
   const [participants, setParticipants] = useState<Participant[]>([]);
   const [audioTracks, setAudioTracks] = useState<AudioTrack[]>([]);
 
-  const sortFunc = options?.sortParticipants ?? sortParticipants;
-
   const connectFn = useCallback(
     async (url: string, token: string, options?: ConnectOptions) => {
```

Comment:
I think the `ConnectOptions` here would become `RoomConnectOptions` it's a subset of the old connect options.